service/frontend/workflowHandler.go

```diff
@@ -590,6 +592,19 @@ func (wh *WorkflowHandler) GetWorkflowExecutionHistory(ctx context.Context, requ
 		continuationToken.IsWorkflowRunning = isWorkflowRunning
 		continuationToken.PersistenceToken = nil
 	}
+
```

Comment:
Can you list potential functions that might return `*serviceerror.DataLoss` here. And also some sort of TODO comment to move this close to history store.